packages/react-scripts/scripts/start.js

```diff
@@ -106,11 +106,13 @@ checkBrowsers(paths.appPath)
       if (err) {
         return console.log(err);
       }
-      if (isInteractive) {
+      if (isInteractive && !process.env.NO_CLEAR_CONSOLE) {
         clearConsole();
       }
       console.log(chalk.cyan('Starting the development server...\n'));
-      openBrowser(urls.localUrlForBrowser);
+      if (!process.env.NO_OPEN_BROWSER) {
```

Comment:
i think you can set env variable `BROWSER=none` for this? https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#advanced-configuration

Reply:
wow, didn't know that, thanks! 😄